### PR TITLE
Simplify removeAttachedFiles function

### PR DESCRIPTION
**Pull request by SimePel**

### CODE REVIEW

File: pkg/pdfcpu/attach.go [hunk 1]
```diff
@@ -197,57 +197,55 @@ func removeAttachedFiles(xRefTable *XRefTable, files StringSet) (ok bool, err er
...
+	if len(files) == 0 {
+		err = xRefTable.RemoveEmbeddedFilesNameTree()
+		if err != nil {
+			return false, err
+		}
+		return true, nil
+	}
 
-		for fileName := range files {
+	removed := false
```
Comment (hhrutter) [inline pkg/pdfcpu/attach.go]: var removed bool .. initializes to false
I would like to keep this here.

Comment (SimePel) [reply pkg/pdfcpu/attach.go]: Ok, as you wish!

**Comment by SimePel** [comment]
I understood. I will open this PR later.

**Comment by hhrutter** [comment]
Thanks 💚

**Pull request merged**